tools/ui-components/src/normalize.css

```diff
@@ -39,6 +39,14 @@ template {
 }
 a {
   background-color: transparent;
+  text-decoration: underline;
```

Comment:
Just a heads up. Normalize should mirror the normalization and element styles of the minimized bootstrap file injected in the head of learn.

This change could go in since we are nearing a migration.